src/utils/validation.ts

```diff
@@ -132,14 +133,22 @@ const LIB_PHONE_NUMBER_URL =
 const loadPhoneValidator = () => dynamicImport(LIB_PHONE_NUMBER_URL);
 
 const validators = {
-  email: (a: any) => emailPattern.test(a),
-  phone: (a: any) => {
+  email: (a: string) => emailPattern.test(a),
+  phone: (a: string) => {
     try {
       return global.libphonenumber.isValidPhoneNumber(a, 'US');
     } catch (e) {
       // Invalid phone number
       return false;
     }
+  },
+  internationalPhone: (a: string) => {
```

Comment:
Could we combine phone and international phone into one validator and just try both formats?  It seems 'phone' is only now used if set in 'login' metadata as a method (?) but it should work there for a common validation.

Reply:
will be refactored later